src/components/modeler/Modeler.vue

```diff
@@ -140,6 +140,8 @@
         @save-state="pushToUndoStack"
       />
     </b-row>
+
+    <RemoteCursor cursor-color="#6CC551" username="Samantha B." />
```

Comment:
I'm guessing this is only for test right?

Reply:
Yes, that's right